README.md

```diff
@@ -21,19 +21,19 @@ def f(x, beta):
 interactive_plot(f, x=x, beta=beta)
 ```
 
-While this will work with any backend except for `%matplotlib inline` you should try to use these with `%matplotlib ipympl` which uses [ipympl](https://github.com/matplotlib/ipympl) as this will have the best performance and UX in a notebook.
+Except for `%matplotlib inline` this will work with any backend. But, if you want to have the best performance and UX in a notebook, you should try to use these with `%matplotlib ipympl` which uses the Matplotlib jupyter extension [ipympl](https://github.com/matplotlib/ipympl).
```

Comment:
It would be cool if we could find a way to emphasize ipympl more. This library is really designed to work with that backend, to the point that for a while I had named it `ipympl-interactions`. Maybe just swapping the ordering would accomplish that. Something like:
```suggestion
These functions are designed to be used with the [ipympl](https://github.com/matplotlib/ipympl), the backend that is designed for use in Jupyter notebooks. So for optimal performance make sure you set the backend with `%matplotlib ipympl`. However, these functions will also work with any interactive backend (e.g. `%matplotlib qt5`).
```